hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -63,10 +63,7 @@ public String showArchivedCommits(
       throws IOException {
     System.out.println("===============> Showing only " + limit + " archived commits <===============");
     String basePath = HoodieCLI.getTableMetaClient().getBasePath();
-    Path archivePath = new Path(basePath + "/.hoodie/.commits_.archive*");
-    if (folder != null && !folder.isEmpty()) {
-      archivePath = new Path(basePath + "/.hoodie/" + folder);
-    }
+    Path archivePath = new Path(HoodieCLI.getTableMetaClient().getArchivePath() + "/.commits_.archive*");
```

Comment:
I'm assuming `HoodieCLI.getTableMetaClient().getArchivePath()` returns the path with `.hoodie` ?

Reply:
```
public String getArchivePath() {
  String archiveFolder = tableConfig.getArchivelogFolder();
  if (archiveFolder.equals(HoodieTableConfig.DEFAULT_ARCHIVELOG_FOLDER)) {
    return getMetaPath();
  } else {
    return getMetaPath() + "/" + archiveFolder;
  }
}
```

Reply:
> I'm assuming `HoodieCLI.getTableMetaClient().getArchivePath()` returns the path with `.hoodie` ?

Yes, it read from metadata and include `.hoodie`.